crates/pathfinder/src/bin/pathfinder/main.rs

```diff
@@ -368,28 +365,28 @@ mod pathfinder_context {
                     network_id: ChainId::MAINNET,
                     gateway: GatewayClient::mainnet(),
                     database: data_directory.join("mainnet.sqlite"),
-                    l1_core_address: Self::MAINNET_CORE,
+                    l1_core_address: EthereumAddress(H160::from(core_addr::MAINNET)),
```

Comment:
Maybe its worth moving these definitions to `EthereumAddress`? e.g. `EthereumAddress::MAINNET` etc. This would require moving them `common` though as otherwise I expect there will be a type cycle or two.. Just a thought, not a strong opinion either way.

Reply:
They won't be `const` anyway. But I think this is a good place to get rid of `EthereumAddress` wrapper.